src/components/table/table.tsx

```diff
@@ -396,6 +396,9 @@ export default function EnhancedTable<
                   >
                     <TableCell component="th" id={labelId} scope="row">
                       {row.first_name}
+                      {!isStudentWithIep(row) &&
+                        Object.keys(row).length < 9 &&
+                        " (me)"}
```

Comment:
Would you please document in a comment that this is a conditional based on the length of the para row vs to case manager row, since that is the only distinction?

Reply:
![Screenshot 2023-11-14 at 5 11 25 PM](https://github.com/sfbrigade/compass/assets/104481165/434ff5cc-80a3-4d1f-bcc9-43db140059e8)